### ISSUE TITLE
Link - add title property

### ISSUE THREAD

**Issue by pankajpandey01** [SAP/ui5-webcomponents-react#5024]
### Is your feature request related to a problem?

This feature request is related to a11y requirement where user need to be informed about the link which will open in window/tab

### Describe the solution you'd like

Title property should be supported where we can provide extra info to user (in our case "info about link opening in new window").  This will help visual user as well as screen reader user.

For example:
```
<Link title="Link open in new window/tab">
  Link Text
</Link>
```

### Additional Context

Organization: {SuccessFactors}

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by MarcusNotheis** [comment]
Hi @pankajpandey01,

this feature is already supported as the Link component (like almost every other component in this project) is accepting all standard HTML attributes which includes `title` as well.

Example CodeSandBox: https://codesandbox.io/p/sandbox/wcr-link-title-qgwntm?file=/src/App.tsx:1,24

**Comment by pankajpandey01** [comment]
Hi @MarcusNotheis,
title attribute added on <ui5-link> is not getting added to inner <a> tag. So screen reader JAWS 2023 not announcing it.
![Screenshot 2023-08-30 at 10 28 18 AM](https://github.com/SAP/ui5-webcomponents-react/assets/67934622/e42025d0-979c-4128-8724-b6ac9a07221a)

**Comment by MarcusNotheis** [comment]
Thanks for the explanation, I didn't get that this issue was about Screen Readers from the initial description. As the Link component is developed in the UI5 Web Components repository, I'll now forward this issue to their repository.
